client/state/current-user/email-verification/actions.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_STATE_RESET,
+} from 'state/action-types';
+
+export function verifyEmail() {
+	return { type: EMAIL_VERIFY_REQUEST };
+}
```

Comment:
Sure, I don't really have a preference :) Updated in cafeb682abd725506be132b4c31602028283efbb